include/solvers/navier_stokes_base.h

```diff
@@ -812,6 +822,29 @@ class NavierStokesBase : public PhysicsSolver<VectorType>
 
   // Member variables
 protected:
+  /**
+   * @brief Verify consistency of the input parameters for boundary
+   * conditions to ensure that for every boundary condition within the
+   * triangulation, a boundary condition has been specified in the input file.
+   */
+  void
+  verify_consistency_of_boundary_conditions()
```

Comment:
Yeah that's a good point. The issue is that the function would have to take two arguments (the type of exceptions and the boundary objects). In the end, it would not be much leaner this way. This can be refactored afterwards though